src/dx/subscribe/event_engine/effects/mod.rs

```diff
@@ -6,9 +6,11 @@ use crate::{
 };
 
 use super::effect_handler::HandshakeFunction;
+use super::ReceiveFunction;
```

Comment:
Should we make it look closer to the way how we use `HandshakeFunction` which used through the `effect_handler` module?